source/installing-splunk/splunk-app.rst

```diff
@@ -67,4 +67,71 @@ Installation
     :align: center
     :width: 100%
 
-Now that you've finished installing Splunk app for Wazuh, you can install and setup Splunk forwarders on the :ref:`next section <splunk_forwarder>`.
+Now that you've finished installing Splunk app for Wazuh, you can install and setup Splunk forwarders on the :ref:`Splunk forwarder section <splunk_forwarder>`.
+
+Installing the Wazuh App in a Splunk cluster
+--------------------------------------------
+
+.. note::
+  We can install the App in each search-head by hand, but if we have hundreds or even thousands of search-heads, it will be better to install it automatically.
+  For this purpose, we are using the **deployer**, a machine that installs the App in every search-head at the same time and automatically.
+
+
+.. warning::
+  We need to eliminate "SplunkAppForWazuh/default/indexes.conf" so it does not create automatic indexes in the search-heads.
+
+After installing the App following the **Official installation guide** in our **deployer** machine, we follow this steps:
+
+.. code-block:: console
+
+  // Copy the app into the splunk cluster folder:
+  # cp -r installation_path/SplunkAppForWazuh /opt/splunk/etc/shcluster/apps
+  // Delete the indexes.conf to don't install automatic indexers:
+  # rm /opt/splunk/etc/shcluster/apps/SplunkAppForWazuh/default/indexes.conf
+  // Create the configuration file we are really using to configure the cluster:
```

Comment:
The `outputs.conf` file in the app is _not_ for configuring any cluster. That file is for sending events to the Indexers peers, in our case, for sending information about the `agents status monitoring`.